client/src/components/R&R/StarFilter.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ProgressBar from './ProgressBar';
+
+export default function StarFilter({stars}) {
+  return (
+    <div className="rr-star-filter">
+      <div className="rr-star-filter-child rr-n-stars">
+        {`${stars} star${stars > 1 ? 's' : ' '}`}
+      </div>
+      <ProgressBar percent={50} />
+      <div className="rr-star-filter-child">
+        500
```

Comment:
what is the 500 for? will it be the amount of ratings for that amount of stars?

Reply:
Yes.